internal/pkg_web/BUILD.bazel

```diff
@@ -31,9 +31,6 @@ nodejs_binary(
 jasmine_node_test(
     name = "assembler_test",
     srcs = ["assembler_spec.js"],
```

Comment:
wow, we already had the test coverage for this bug too? :(
We ought to take a pass over all the fix-windows tagged targets...

Reply:
Yeah that’s a good idea. I was also surprised to see that there was coverage.